content/en/synthetics/api_tests/icmp_tests.md

```diff
@@ -0,0 +1,163 @@
+
+| Error             | Description                                                                                                                                                                                                                                                                                                                                                                                                                                            |
+|-------------------|--------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------|
+| `CONNRESET`       | The connection was abruptly closed by the remote server. Possible causes include the webserver encountering an error or crashing while responding, or loss of connectivity of the webserver.                                                                                                                                                                                                                                                         |
+| DNS               | DNS entry not found for the test URL. Possible causes include misconfigured test URL, wrong configuration of your DNS entries, etc.                                                                                                                                                                                                                                                                                                                  |
+| `INVALID_REQUEST` | The configuration of the test is invalid (for example, a typo in the URL).                                                                                                                                                                                                                                                                                                                                                                                     |
+| `SSL`             | The SSL connection couldn't be performed. [See the dedicated error page for more information][8].                                                                                                                                                                                                                                                                                                                                                      |
```

Comment:
Let's completely get rid of the SSL line - I recently removed it on DNS & TCP tests :)